docs/en/operation/StressBench.md

```diff
@@ -111,3 +111,69 @@ This process continues until the max throughput is discovered.
 This algorithm uses many iterations and time to determine the throughput, but it produces a more consistent result, and the actual number is meaningful.
 If the resulting throughput is X, that means if the clients produce X throughput, the server can achieve that.
 It also means if the clients produce more than X throughput, like X+N, the server will not be able to achieve X+N throughput.
+
+
+## Fuse IO Stress Bench
+The Fuse IO Stress Bench is a tool to measure the reading performance of Alluxio Fuse-based POSIX API.
+
+### Procedure
+
+#### 0. Prerequisite
+A running Alluxio cluster is required. The Alluxio path for testing needs to be mounted to the local file system. To prevent Fuse cache affecting
+test accuracy, it is suggested to use different mount points for writing test files and read. See 
+[FUSE-based POSIX API](https://docs.alluxio.io/os/user/stable/en/api/POSIX-API.html) for details on how to mount Alluxio to local file system.
+
+#### 1. Write test files
+Write the test files by running the benchmark with `--operation Write` into the mount point for writing files. More specifically, run 
+
+`$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench --operation Write --local-path /path/to/writing/mount/point ...`
```

Comment:
Always use three backstick ```
for a paragraph of command

Reply:
same for other places

Reply:
It will be better to not using ...
if ... is needed, add a detailed example afterwards

In your case, i will prefer to have two parts
- one general usage for each write/list/read case
- one detailed example for each write/list/read case

`$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench --operation Write --local-path /path/to/writing/mount/point ...`
For example
`.....`